tests/kernel/common/src/irq_offload.c

```diff
@@ -148,7 +148,15 @@ __no_optimization void test_nop(void)
 	arch_nop();
 	arch_nop();
 	arch_nop();
-#elif defined(CONFIG_ARMV8_A)
+#endif
+	t_after = k_cycle_get_32();
+
+	/* Calculate delta time of arch_nop(). */
+	diff = t_after - t_before - t_get_time;
+	printk("arch_nop() takes %d cycles\n", diff);
+
```

Comment:
why have you moved this block ? That is not necessary if you are skipping the test.

Reply:
Thanks for your review! @ceolin . At first, my idea is that we still try to measure spend cycles even if skipping them, but it seems not very meaningful as you mentioned. I will change it back.